docs/getting-started/integrations/drupal-real-time-collaboration.md

```diff
@@ -41,9 +41,9 @@ You can use all the collaboration features **for {@link features/collaboration#a
 
 Apart from the real-time collaboration tools, the module provides various other plugins. The {@link features/format-painter format painter} feature lets users consistently style the edited text, while the {@link features/mentions mentions} feature allows you to tag other users in comments. Meanwhile, {@link features/slash-commands slash commands} let you create, insert, and format rich content on the go by typing the `/` character and choosing from many predefined actions, such as text formatting, and inserting headings, tables, or lists.
 
-### Full-screen mode
+### Fullscreen mode
```

Comment:
IIRC, fullscreen for Drupal is a separate Drupal module that we created, dedicated for Drupal. After we release the official plugin, we need to make decision what do we do with the module. I don't know whether it offers anything beyond what we will offer in the official plugin, so I wouldn't be surprised if we depracte/remove it. /cc @andrzejkala